src/wallet/components/WalletModal.tsx

```diff
@@ -0,0 +1,261 @@
+                  alt={`${appName || 'App'} icon`}
+                  className="h-full w-full object-cover"
+                />
+              </div>
+            )}
+            {appName && (
+              <h2 className={cn(text.headline, color.foreground)}>{appName}</h2>
+            )}
+          </div>
+        )}
+
+        <div className="flex flex-col gap-3">
+          <button
+            type="button"
+            onClick={handleSmartWalletConnector}
+            className={cn(
+              border.radiusInner,
+              line.default,
+              text.label2,
+              pressable.alternate,
+              color.foreground,
+              'h-10 w-[275px] px-4 py-2.5',
+              'flex items-center justify-between text-left',
+            )}
+          >
+            Sign up
+            <div className="h-4 w-4">{defaultAvatarSVG}</div>
+          </button>
+
+          <div className="relative">
+            <div className="absolute inset-0 flex items-center">
+              <div className={cn(line.default, 'w-full border-[0.5px]')} />
+            </div>
+            <div className="relative flex justify-center">
+              <span
+                className={cn(
+                  background.default,
+                  color.foregroundMuted,
+                  text.legal,
+                  'px-2',
+                )}
+              >
+                or continue with an existing wallet
+              </span>
+            </div>
+          </div>
+
+          <button
+            type="button"
+            onClick={handleCoinbaseWalletConnector}
+            className={cn(
+              border.default,
+              border.radiusInner,
+              line.default,
+              text.label2,
+              pressable.alternate,
+              color.foreground,
+              'h-[40px] w-[275px] px-4 py-2.5',
+              'flex items-center justify-between text-left',
+            )}
+          >
+            Coinbase Wallet
+            <div className="h-4 w-4">{coinbaseWalletSvg}</div>
+          </button>
+
+          <button
+            type="button"
+            onClick={handleWalletConnectConnector}
+            className={cn(
+              border.default,
+              border.radiusInner,
+              line.default,
+              text.label2,
+              pressable.alternate,
+              color.foreground,
+              'flex h-[40px] w-[275px] px-4 py-2.5',
+              'items-center justify-between text-left',
+            )}
+          >
+            Other wallets
+            <div className="h-4 w-4">{walletConnectSvg}</div>
+          </button>
+        </div>
+
+        <div
+          className={cn(
+            color.foregroundMuted,
+            text.legal,
+            'flex flex-col items-center justify-center gap-1 px-4',
+            'mt-4 w-[275px] text-center leading-3',
+          )}
+        >
+          <span className="font-normal text-[10px] leading-[13px]">
```

Comment:
Agreed, would love if our linter could enforce this

Reply:
I can spend some time looking at the linter, but Biome generally has poor support for linting/formatting Tailwind.

But do we use text-10px frequently? If so, we can add it to the Tailwind spacing system so that it can be used across the codebase

cc @mindapivessa